tests/default.nf.test

```diff
@@ -15,20 +15,136 @@ nextflow_pipeline {
+                  { assert snapshot (
+                        stable_name_kraken2, stable_content_kraken2, file ("$outputDir/kraken2/db2/2613_db2.kraken2.kraken2.report.txt").text.contains("S288C"), file ("$outputDir/kraken2/db2/2611_db2.kraken2.kraken2.report.txt").text.contains("S288C"), file ("$outputDir/kraken2/db2/2612_db2.kraken2.kraken2.report.txt").text.contains("S288C"),
+                        file("$outputDir/kraken2/db2/2611_db2.kraken2.classified.fastq.gz").text.contains(">ERR5766174.15494"),file("$outputDir/kraken2/db2/2612_db2.kraken2.classified.fastq.gz").text.contains("@ERR5766180.35963"),file("$outputDir/kraken2/db2/2613_db2.kraken2.classified.fastq.gz").text.contains("@ERR5766181.6866"))
+                        .match("kraken2") },
+                { assert snapshot ( stable_name_bowtie2, file ("$outputDir/bowtie2/align/2613_ERR5766181.bowtie2.log").text.contains("overall alignment rate"), file ("$outputDir/bowtie2/align/2612_ERR5766176_B.bowtie2.log").text.contains("overall alignment rate"), file ("$outputDir/bowtie2/align/2612_ERR5766176.bowtie2.log").text.contains("overall alignment rate"), file ("$outputDir/bowtie2/align/2612_ERR5766180.bowtie2.log").text.contains("overall alignment rate"),
+                        path("$outputDir/bowtie2/align/"))
+                        .match("bowtie2" ) },
+                { assert snapshot ( stable_name_bracken, file ("$outputDir/bracken/db1/2613_db1.bracken.tsv").text.contains("Homo sapiens"), file ("$outputDir/bracken/db1/2611_db1.bracken.tsv").text.contains("Homo sapiens"), file ("$outputDir/bracken/db1/2612_db1.bracken.tsv").text.contains("Homo sapiens"),
+                        path("$outputDir/bracken/db1"))
+                        .match("bracken") },
+                { assert snapshot ( stable_name_centrifuge, file ("$outputDir/centrifuge/db3/2613_db3.centrifuge.report.txt").text.contains("Eucommia"), file ("$outputDir/centrifuge/db3/2612_db3.centrifuge.report.txt").text.contains("Eucommia"),
+                        stable_content_centrifuge)
+                        .match("centrifuge") },
+                { assert snapshot ( file ("$outputDir/fastp/2613_ERR5766181.fastp.log").text.contains("Read pairs merged"),file ("$outputDir/fastp/2612_ERR5766176_B.fastp.log").text.contains("Read pairs merged"),file ("$outputDir/fastp/2612_ERR5766180.fastp.log").text.contains("Read pairs merged"),stable_content_fastp, stable_name_fastp ).match("fastp") },
+                { assert snapshot ( stable_name_ganon, stable_content_ganon, file ("$outputDir/ganon/db1/2613_db1.ganon.tre").text.contains("Saccharomycetales"),file ("$outputDir/ganon/db1/2612_db1.ganon.tre").text.contains("Saccharomycetales"), file ("$outputDir/ganon/db1/2611_db1.ganon.tre").text.contains("Saccharomycetales")).match("ganon") },
```

Comment:
Why are you checkign te `tre` when files when you've theroretically not excluded them from storing md5sums - I also don't see md5sums within the snapshot which is confusing 🤔